tests/test_result.py

```diff
@@ -213,6 +213,26 @@ def test_and_then() -> None:
     assert Err(3).and_then(sq_lambda).and_then(sq_lambda).err() == 3
 
 
+def test_inspect() -> None:
+    oks: List[int] = []
+    assert Ok(2).inspect(lambda x: oks.append(x)) == Ok(2)
```

Comment:
Can you add lines to test using a regular function as well, you'll discover the issue of `None` vs `Any` as I mentioned in my earlier comment,

```python
# Move to bottom of this file along side the other similar functions
def concat_str(x: str) -> str: 
    return x + x

def test_inspect() -> None:
    Err("e").inspect_err(concat_str)
    Err("e").inspect(concat_str)
```

Reply:
added a test with a regular function